reference/api/indexes.md

```diff
@@ -64,15 +64,19 @@ Get information about an [index](/learn/core_concepts/indexes.md).
 
 <RouteHighlighter method="POST" route="/indexes"/>
 
-Create an [index](/learn/core_concepts/indexes.md).
+Create an [index](/learn/core_concepts/indexes.md). This route accepts two arguments: `index_uid` and `primaryKey`.
 
-This route takes as parameter an unique `uid` and **optionally** the [primary key](/learn/core_concepts/indexes.md#primary-key).
+`index_uid` is mandatory. It should be a unique string identifying your new index.
+
+`primaryKey` is optional and indicates the field containing your documents' [primary key](/learn/core_concepts/indexes.md#primary-key). If you do not supply a value for `primaryKey`, MeiliSearch will do its best to infer it once you add your first document to the index.
```

Comment:
Personally I'm not quite convinced that these paragraphs add enough value to justify the screen real estate; we already have the `Body` table below. Instead of ballooning the description of the endpoint could we incorporate whatever information isn't already present into that table?

(FWIW, in the long run, I am anti-table and hoping to transform a lot of these into paragraphs, much like [Stripe options](https://stripe.com/docs/js/elements_object/create#stripe_elements-options))